packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -81,7 +81,10 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
       overflowItems: [...overflowItems!],
       minimumOverflowItems: [...overflowItems!].length, // for tracking
       farItems,
-      cacheKey: '',
+      cacheKey: this._computeCacheKey({
+        primaryItems: [...items],
```

Comment:
Does items get mutated by _computeCacheKey? If not it seems like you could pass items in without expanding it.